Opt "About Me" into typography styles piecemeal

john-kurkowski opened the pull request:
Previously, with global styles, the custom "Experience" section of the "About Me" page was riddled with inconsistencies with the upstream resume, in presentation and code. Applying typography styles piecemeal allows the "About Me" page to better stay in sync.

# Changes

* Remove global paragraph style
* Remove global style undo hack
* Extract custom "Experience" section from Markdown to own component
* Sync from upstream resume
* Wrap Markdown content that should have the old styles in `<div class="post">`

john-kurkowski committed:
Opt "About Me" into typography styles piecemeal

* Remove global paragraph style
* Remove global style undo hack
* Sync from upstream resume
* Wrap Markdown content that should have the old styles in `<div
  class="post">`

Previously, with global styles, the custom "Experience" section of the
"About Me" page was riddled with inconsistencies with the upstream
resume, in presentation and code. Applying typography styles piecemeal
allows the "About Me" page to better stay in sync.

john-kurkowski commented on src/pages/about.mdx:
```diff
@@ -6,6 +6,8 @@ title: 'About Me'
 import Experience from '../components/resume/experience.astro'
 import Time from '../components/resume/time.astro'
 
+<div class="post">
```
Scattering these `<div>`s is clumsy though. I wonder if that's the trade-off to accept, or if there's a better way.

john-kurkowski replied:
There is an eject button, to override whenever Markdown encounters standard tags like `p` with a custom component. See [this example](https://lukapeharda.com/article/using-tailwindcss-classes-in-markdown/) with Tailwind classes in Astro.

It'd be a lot at the moment to port the global typography styles to that new way of doing things. So I'll stick with this line's CSS cascade for now.

john-kurkowski committed:
fixup! Opt "About Me" into typography styles piecemeal

lost-pixel-signals commented:
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

john-kurkowski committed:
Extract custom HTML from Markdown to own component

Highlights this chunk of the "About Me" page as different. Removes the
need to escape the content as not Markdown, i.e. surround in curly braces.

lost-pixel-signals commented:
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ page/about__[w1280px]*: 


| Before | After |
| --- | --- |
| ![](https://lp-00001-a8c7a.fra1.digitaloceanspaces.com/clud602ae10romo0e861bvpv2/38f538f4181d5d6cfa45d3a62f30758c4d184403b36d145941b7fe4ec6039ba4.png) | ![](https://lp-00001-a8c7a.fra1.digitaloceanspaces.com/clud602ae10romo0e861bvpv2/c5ac266d1384ea7630785d13c0300c49386cb9dc32602440493ae3047f9c678d.png) |



*❗ page/about__[w414px]*: 


| Before | After |
| --- | --- |
| ![](https://lp-00002-d34ac.fra1.digitaloceanspaces.com/clud602ae10romo0e861bvpv2/305984e6c278b777c39ae46be15eff1aea28fc1fc84b43563b53f60a1a715916.png) | ![](https://lp-00001-a8c7a.fra1.digitaloceanspaces.com/clud602ae10romo0e861bvpv2/e4d9d3506fd6bf5b96220e6d43881e56a0bfce309277248dc1925dfb348babaf.png) |

john-kurkowski committed:
Work around Astro preserving newlines

"Astro is not like JSX." This Prettier-style wrapping generated
unwanted additional link underline.

lost-pixel-signals commented:
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |
